director/director_api.go

```diff
@@ -33,3 +49,78 @@ func ListNamespacesFromOrigins() []NamespaceAd {
 	}
 	return namespaces
 }
+
+func LoadDirectorPublicKey() (*jwk.Key, error) {
```

Comment:
I don't see that this function is called anywhere. Should something be hooked into it? Maybe in the jwks handler logic from `director/authentication.go`?

Reply:
This is a helper function for origin/other service to call to obtain the public key from the director. It won't be called anywhere in this PR but will be used for the following director-based health test PR.

Reply:
Ahh, gotcha.